srcpkgs/gmsl/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gmsl'
+pkgname=gmsl
+version=1.1.8
+revision=1
+noarch=yes
+create_wrksrc=true
+depends="make"
+short_desc="GNU Make Standard Library"
+maintainer="B. Wilson <x@wilsonb.com>"
+license="BSD-3-Clause-Clear"
+homepage="https://sourceforge.net/projects/gmsl/"
+distfiles="${SOURCEFORGE_SITE}/gmsl/${pkgname}-${version}.tar.gz"
```

Comment:
write out gmsl instead of using ${pkgname}

Reply:
May I ask for a reasoning? There are tons of templates that contain `${pkgname}` in `distfiles`.